iceoryx_hoofs/test/moduletests/test_posix_mutex.cpp

```diff
@@ -222,7 +222,10 @@ TEST_F(Mutex_test, MutexWithDeadlockDetectionsFailsWhenAnotherThreadTriesToUnloc
 TEST_F(Mutex_test,
        MutexWithOnReleaseWhenLockedBehaviorUnlocksLockedMutexWhenThreadTerminatesAndSetsItIntoInconsistentState)
 {
-    ::testing::Test::RecordProperty("TEST_ID", "4da7b1fb-23f1-421c-acf3-2a3d9e26b1a1");
+#if defined(QNX) || defined(__QNX) || defined(__QNX__) || defined(QNX__)
+    GTEST_SKIP() << "iox-#1683 QNX supports robust mutex not like the posix standard describes them."
+#endif
+        ::testing::Test::RecordProperty("TEST_ID", "4da7b1fb-23f1-421c-acf3-2a3d9e26b1a1");
```

Comment:
@elfenpiff  The `RecordProperty` must be called before the GTEST_SKIP macro (outside of the ifdef)

CC @dongilchang